lib/assayer.js

```diff
@@ -1,49 +1,42 @@
 'use strict';
 
-const useragent = require('./assays/useragent');
 const domainthreat = require('./assays/domainthreat');
 const datacenter = require('./assays/datacenter');
 
 /**
  * Determine if a download is a duplicate, and return metadata.
  */
 exports.test = async record => {
-  const [agent, threat, center] = await Promise.all([
-    useragent.look(record.remoteAgent),
+  const [threat, center] = await Promise.all([
     domainthreat.look(record.remoteReferrer),
     datacenter.look(record.remoteIp),
   ]);
   if (record.download && record.download.isDuplicate) {
-    return { isDuplicate: true, cause: record.download.cause || 'unknown', agent };
-  } else if (agent.bot) {
-    return { isDuplicate: true, cause: 'bot', agent };
```

Comment:
wait, so we won't have bots labelled anymore? are those just not getting sent through now?

Reply:
Just clarifying from a slack convo:

Bots are still passed through right now.  But DTR marks them as bots, not this lambda.